app/assets/scripts/map.js

```diff
@@ -94,6 +112,23 @@ function displayDetailedResourceView(marker) {
   });
 }
 
+var submitReview = function(rating, review, id){
```

Comment:
let's keep the style for declaring functions consistent
`function submitReview(rating, review, id) {`